### PR TITLE
Pick a specific instance of InstancedMesh

### PR DESCRIPTION
**Pull request by lej0hn**
Hey everybody!
I was facing an issue described in #2615 and I came up with this example to solve it (borrowing from previous ones), which also demonstrates a bit how the get_transform() is used to translate from visual coordinate system to the canvas one. It just prints out the particular instance position clicked and some additional info

### CODE REVIEW

**Comment by brisvag** [comment]
Love it, thanks for the PR! Seems to be working fine. Let's see if CI is happy!

**Comment by lej0hn** [comment]
I'm not sure to what the unsuccessful check refers to, can I do something about it?

File: examples/scene/instanced_mesh_picking.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+Picking Instance of InstancedMeshes
+=========================
```
[the remark this answers is not in the document]
Comment (djhoese) [reply examples/scene/instanced_mesh_picking.py]: > There website errors are always a mystery to me, I never can find the error in the logs

The errors are never at the bottom or near the bottom of the logs. That's the problem. You have to look at each block of the log separately or on a large monitor to make it obvious where the warning/error is. Looks like the last commit fixed it!

**Review by brisvag — APPROVED** [review]

**Comment by brisvag** [comment]
Alright, looks good, let's get this in! Thanks @lej0hn :)

**Pull request merged**